### PR TITLE
Azure multiple env config

### PR DESCRIPTION
**Pull request by KrzychuR**

### CODE REVIEW

File: BLocal/packages.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="6.0.5" targetFramework="net35" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net35" />
```
Comment (rusln) [inline BLocal/packages.config]: @KrzychuR the should it be: `net35` -> `net40`?

File: BLocal/BLocal.csproj [hunk 2]
```diff
@@ -62,8 +62,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\Shared.Management.Objects.for.SQL.Server.2008.R2.11.0.3000.0\lib\Microsoft.SqlServer.SqlEnum.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.6.0.5\lib\net35\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (rusln) [inline BLocal/BLocal.csproj]: @KrzychuR should it be: `Version=9.0.0.0` -> `Version=9.0.1` ?

File: BLocal.Web.Manager.MVC3/Web.config.template [hunk 3]
```diff
@@ -26,9 +26,9 @@
     <add name="iChoosr - Test" color="#f90">
       <valueProvider type="BLocal.Web.Manager.Providers.RemoteAccess.RemoteAccessManager, BLocal.Web.Manager, Version=1.0.0.0, Culture=neutral">
         <!-- Add constructor parameters -->
-        <add name="targetUrl" value="http://10.120.3.199:7000/ExternalSynchronization/" />
-        <add name="targetPassword" value="left-out-for-safety"/>
-        <add name="targetProviderPair" value="iChoosr - Test"/>
+        <add name="targetUrl" value="http://10.120.3.199:7000/RemoteAccess/" />
```
Comment (rusln) [inline BLocal.Web.Manager.MVC3/Web.config.template]: @KrzychuR ExternalSynchronization -> RemoteAccess & targetProviderPair -> targetProviderGroup. Shouldn't there also be un update to code for these changes to work?

Comment (KrzychuR) [reply BLocal.Web.Manager.MVC3/Web.config.template]: @rusln I didn't found /ExternalSynchronization endpoint on our prod/test env. I believe it is mistake/old config on Git :(

File: BLocal/packages.config (hunk 1 above)
Comment (KrzychuR) [reply BLocal/packages.config]: @rusln I didn't test BLocal with new .NET maybe we should stay with 3.5?

File: BLocal/BLocal.csproj (hunk 2 above)
Comment (KrzychuR) [reply BLocal/BLocal.csproj]: @rusln I have no idea why VS/NuGet added 9.0.0.0 with reference to ..\packages\Newtonsoft.Json.9.0.1\lib\net35\Newtonsoft.Json.dll :) I've used NuGet Package Manager for solution and package manager reporting version 9.0.0.1 :(

**Pull request merged**